dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -121,10 +120,11 @@ janino/3.0.16//janino-3.0.16.jar
 javassist/3.25.0-GA//javassist-3.25.0-GA.jar
 javax.jdo/3.2.0-m3//javax.jdo-3.2.0-m3.jar
 javolution/5.5.1//javolution-5.5.1.jar
+jaxb-api/2.2.11//jaxb-api-2.2.11.jar
 jaxb-runtime/2.3.2//jaxb-runtime-2.3.2.jar
 jcl-over-slf4j/1.7.32//jcl-over-slf4j-1.7.32.jar
 jdo-api/3.0.1//jdo-api-3.0.1.jar
-jdom/1.1//jdom-1.1.jar
+jdom2/2.0.6//jdom2-2.0.6.jar
```

Comment:
I'm checking licenses of these too - this is basically 3-clause BSD and could be listed as such in the LICENSE/NOTICE file with other 3-clause BSD. I think we might need the license file stored in licenses/ - https://raw.githubusercontent.com/hunterhacker/jdom/master/LICENSE.txt

Reply:
Updated. Can you check if it looks OK?